src/test/clojure/xtdb/indexer_test.clj

```diff
@@ -88,7 +88,7 @@
           last-tx-key (serde/->TxKey magic-last-tx-id (time/->instant #inst "2020-01-02"))]
       (util/delete-dir node-dir)
 
-      (util/with-open [node (tu/->local-node {:node-dir node-dir})]
+      (util/with-open [node (tu/->local-node {:node-dir node-dir :compactor-threads 0})]
         (let [db (db/primary-db node)
```

Comment:
I am turning the compactor off here, so that ordering in the partitions is deterministic.